data.js

```diff
@@ -633,6 +633,14 @@ module.exports = [
     url: "https://github.com/daniellmb/aop.js",
     source: "https://raw.githubusercontent.com/daniellmb/aop.js/master/aop.js"
   },
+  {
+    name: "svelte-system-info",
+    github: "rozek/svelte-system-info",
+    tags: ["browser-detection", "os-detection"],
+    description: "determines name and version of underlying browser and operating system (not only for Svelte)",
+    url: "https://github.com/rozek/svelte-system-info",
+    source: "https://raw.githubusercontent.com/rozek/svelte-system-info/main/src/svelte-system-info.ts"
```

Comment:
From the readme - emphasis on "ready-to-use", which means you should not link to a TypeScript source file:

> The 'source' attribute must point to a single-file, or an array of files that can be concatenated to form your complete library in a ready-to-use form. The 'source' must not be minified (see below)